content/docs/rfcs/13/README.md

```diff
@@ -71,7 +71,8 @@ message ContentFilter {
 
 message HistoryQuery {
   repeated ContentFilter contentFilters = 2;
-  PagingInfo pagingInfo = 3;
+  string pubsubtopic = 3;
```

Comment:
If we are already changing field orders, I wodner if pubsubtopic (should be pubsubTopic or pubSubTopic probably) shouldn't be before contentFilter? It is a "higher order" field

Reply:
Makes sense, will do it.

Reply:
This does not match nim-waku code anymore, are you aware?

Reply:
Right! Thanks!